### PR TITLE
Run aktualizr-info in readonly mode by default

### PR DESCRIPTION
**Pull request by patriotyk**

### CODE REVIEW

File: src/libaktualizr/storage/sqlstorage.cc [hunk 1]
```diff
@@ -1343,6 +1343,11 @@ bool SQLStorage::dbMigrate() {
     return true;
   }
 
+  if (readonly_) {
+    LOG_ERROR << "Dtabase is opened in readonly mode and cannot be migrated to latest version";
```
Comment (lbonn) [inline src/libaktualizr/storage/sqlstorage.cc]: Typo, "Database"

File: src/libaktualizr/storage/sqlstorage.cc [hunk 2]
```diff
@@ -97,7 +97,7 @@ SQLStorage::SQLStorage(const StorageConfig& config) : INvStorage(config) {
 }
 
 void SQLStorage::storePrimaryKeys(const std::string& public_key, const std::string& private_key) {
-  SQLite3Guard db(dbPath());
+  SQLite3Guard db(dbPath(), readonly_);
```
Comment (lbonn) [inline src/libaktualizr/storage/sqlstorage.cc]: Maybe it's time to set up a private helper method, the boiler-plate is starting to get a bit heavy:

```
SQLite3Guard SQLStorage::dbConnection() {
  SQLite3Guard db(dbPath(), readonly_);
  if (db.get_rc() != SQLITE_OK) {
    LOG_ERROR << "Can't open database: " << db.errmsg();
  }
  return std::move(db);
}
```

And usage:

```
  SQLite3Guard db = dbConnection();
  if (db.get_rc() != SQLITE_OK) {
    return false;
  }
```

Comment (patriotyk) [reply src/libaktualizr/storage/sqlstorage.cc]: Nice idea

**Comment by cajun-rat** [comment]
This is great.  Removing all the duplication in SQLStorage is very good, and I like any commit that _reduces_ the number of lines on code in the codebase.

**Pull request merged**